### PR TITLE
Minor doc and db updated.

### PR DESCRIPTION
**Pull request by kaninhot004**
[item_db.txt]
- Correct 2158 item bonus.
- Removed comment code on 19387 item bonus and correct temporary variables name.

[item_combo_db.txt]
- Added missing semicolon.

[item_bonus.txt]
- Added missing RC2.
- Added missing ATF. (Item id: 1674 had it.)
- Removed duplicated bonus.

[status_change.txt]
- Added missing SC.

* **Addressed Issue(s)**: https://github.com/rathena/rathena/issues/4746

* **Server Mode**: Renewal

* **Description of Pull Request**: I had correct up function, Added missing semicolon.

### CODE REVIEW

File: db/re/item_db.txt [hunk 1]
```diff
@@ -10216,7 +10216,7 @@
...
 19381,Protect_Cloth,Protect Cloth,4,20,,300,,0,,0,0xFFFFFFFF,63,2,1,,90,0,1761,{ bonus bMdef,5; bonus bMaxHPrate,5; },{},{}
-19387,Experimental_Goat_Cap,Experimental Goat Cap,4,20,,500,,2,,1,0xFFFFFFFF,63,2,256,,40,1,1768,{ .@r = getrefine(); .@sub = 10; .@dmg = 15; if (.@r >= 7) { .@dmg += 15; if (.@r >= 8) { .@sub += 15; if (.@r >= 9) { .@dmg += 20; } } } bonus2 bSubEle,Ele_Earth,.@val; /*bonus2 bAddRace2,RC2_WERNER_LAB,.@dmg; bonus2 bMagicAddRace2,RC2_WERNER_LAB,.@dmg; bonus2 bSubRace2,RC2_WERNER_LAB,.@dmg; Werner's Central lab is not implemented yet */ },{},{}
+19387,Experimental_Goat_Cap,Experimental Goat Cap,4,20,,500,,2,,1,0xFFFFFFFF,63,2,256,,40,1,1768,{ .@r = getrefine(); .@sub = 10; .@dmg = 15; if(.@r >= 7){ .@dmg += 15; if(.@r >= 8){ .@sub += 15; if(.@r >= 9){ .@dmg += 20; }}} bonus2 bSubEle,Ele_Earth,.@sub; /*bonus2 bAddRace2,RC2_WERNER_LAB,.@dmg; bonus2 bMagicAddRace2,RC2_WERNER_LAB,.@dmg; bonus2 bSubRace2,RC2_WERNER_LAB,.@dmg; Werner's Central lab is not implemented yet */ },{},{}
```
Comment (aleos89) [inline db/re/item_db.txt]:
```suggestion
19387,Experimental_Goat_Cap,Experimental Goat Cap,4,20,,500,,2,,1,0xFFFFFFFF,63,2,256,,40,1,1768,{ .@r = getrefine(); .@sub = 10; .@dmg = 15; if (.@r >= 7) { .@dmg += 15; if (.@r >= 8) { .@sub += 15; if (.@r >= 9) { .@dmg += 20; } } } bonus2 bSubEle,Ele_Earth,.@sub; /*bonus2 bAddRace2,RC2_WERNER_LAB,.@dmg; bonus2 bMagicAddRace2,RC2_WERNER_LAB,.@dmg; bonus2 bSubRace2,RC2_WERNER_LAB,.@dmg; Werner's Central lab is not implemented yet */ },{},{}
```
Just to keep the formatting how it was.

File: doc/item_bonus.txt [hunk 2]
```diff
@@ -58,6 +59,7 @@ This list contains all available constants referenced in the 'bonus' commands.
 	ATF_LONG	= Trigger on ranged attacks
 	
 	Skill/attack type criteria: (Default: Physical/weapon)
+	ATF_SKILL	=
```
Comment (aleos89) [inline doc/item_bonus.txt]:
```suggestion
	ATF_SKILL	= Trigger on magic/misc skills
```

File: doc/item_bonus.txt [hunk 3]
```diff
@@ -274,7 +276,6 @@ Ignore Def
 ----------
 bonus bIgnoreDefEle,e;  		Disregard DEF against enemies of element e
 bonus bIgnoreDefRace,r; 		Disregard DEF against enemies of race r
-bonus2 bIgnoreDefRaceRate,r,n;  	Disregard n% of the target's DEF if the target belongs to race r
```
Comment (aleos89) [inline doc/item_bonus.txt]: Why remove this? It's defined as a valid item bonus:
```cpp
export_constant2("bIgnoreDefRaceRate",SP_IGNORE_DEF_RACE_RATE);
```

File: doc/item_bonus.txt [hunk 4]
```diff
@@ -310,6 +311,7 @@ bonus4 bAddEffWhenHit,eff,n,atf,t;	Adds a n/100% chance to cause status eff for
                               		  ATF_SHORT  = trigger on melee attacks
                               		  ATF_LONG   = trigger on ranged attacks
                               		 Skill/attack type criteria: (Default: Physical/weapon)
+									  ATF_SKILL  =
```
Comment (aleos89) [inline doc/item_bonus.txt]:
```suggestion
                              		  ATF_SKILL  = trigger on magic/misc skills
```

File: doc/item_bonus.txt (hunk 3 above)
Comment (kaninhot004) [reply doc/item_bonus.txt:277]: It's duplicated.
image

File: doc/item_bonus.txt [hunk 5]
```diff
@@ -310,6 +311,7 @@ bonus4 bAddEffWhenHit,eff,n,atf,t;	Adds a n/100% chance to cause status eff for
                               		  ATF_SHORT  = trigger on melee attacks
                               		  ATF_LONG   = trigger on ranged attacks
                               		 Skill/attack type criteria: (Default: Physical/weapon)
+									  ATF_SKILL  = trigger on magic/misc skills
```
Comment (aleos89) [inline doc/item_bonus.txt:314]: The spacing on this one is off because of tabs vs spaces.

File: doc/item_bonus.txt (hunk 4 above)
Comment (kaninhot004) [reply doc/item_bonus.txt]: Thanks! done.

**Pull request merged**